specifications/connector/radar-altoida-connector.yml

```diff
@@ -0,0 +1,37 @@
+#====================================== Altoida Phone Sensors =====================================#
+vendor: Altoida
+model: altoida.app
+version: 1.0.0
+data:
+  #Phone sensors
+  - doc: Data from the accelerometer sensor of 3 axis in G.
+    topic: altoida_accelerometer
+    value_schema: .connector.altoida.ACC
```

Comment:
This should be `.connector.altoida.AltoidaAcceleration` and likewise for the other value schemas. You can mention in the doc for each of those files what the original filename was (ACC, etc.).